src/app/wishlists/wishlists.ts

```diff
@@ -92,8 +92,10 @@ function isWeaponOrArmorOrGhostMod(plug: DimPlug): boolean {
     return false;
   }
 
-  return plug.plugItem.itemCategoryHashes?.some(
-    (ich) => ich === 610365472 || ich === 4104513227 || ich === 303512563 || ich === 4176831154
+  return (
+    plug.plugItem.itemCategoryHashes?.some(
+      (ich) => ich === 610365472 || ich === 4104513227 || ich === 303512563 || ich === 4176831154
+    ) ?? false
```

Comment:
Maybe find the intersection instead?
```
return (
[610365472,4104513227,303512563,4176831154].filter(plug.plugItem.itemCategoryHashes?.includes)
) ?? false
```

Reply:
For just four hashes, it's probably faster to do four inline tests than allocate an array and invoke filter.